src/madness/mra/funcimpl.h

```diff
@@ -4771,84 +4781,185 @@ template<size_t NDIM>
         void do_apply(const opT* op, const keyT& key, const Tensor<R>& c) {
```

Comment:
@robertjharrison much of #563 does not need your eyes, but this (=the whole enchilada) does .. might be more efficiento to walk you through